configs/expose/resnet18_hmr_expose_face.py

```diff
@@ -0,0 +1,183 @@
+_base_ = ['../_base_/default_runtime.py']
+use_adversarial_train = True
+
+# evaluate
+evaluation = dict(interval=10, metric=['3DRMSE'])
+
+optimizer = dict(
+    backbone =  dict(type='Adam', lr=1.0e-4, weight_decay = 1.0e-4),
+    head = dict(type='Adam', lr=1.0e-4, weight_decay = 1.0e-4),
+)
+optimizer_config = dict(grad_clip=None)
+lr_config = dict(policy='step',step=[60, 100],gamma = 0.1)
+
+runner = dict(type='EpochBasedRunner', max_epochs=100)
+
+log_config = dict(
+    interval=50, hooks=[
+        dict(type='TextLoggerHook'),
+    ])
+
+
+checkpoint_config = dict(interval=10)
+
+
+# model settings
+
+find_unused_parameters = True
+
+model = dict(
+    type = 'SMPLXImageBodyModelEstimator',
+    backbone = dict(
+        type = 'ResNet',
+        depth = 18,
+        init_cfg=dict(
+            type='Pretrained',
+            checkpoint='data/pretrained_models/resnet18.pth')
+    ),
+    head = dict(
+        type = 'ExPoseFaceHead',
+        num_betas = 100,
+        num_expression_coeffs = 50,
+        mean_pose_path = 'data/body_models/all_means.pkl',
```

Comment:
Wheter we should change the name of this mean files, it is a little confusing here.

Reply:
ExPoseFaceHead means model head for human face prediction.